README.md

```diff
@@ -1 +1,45 @@
-# Hello-Microverse-Project
\ No newline at end of file
+# Hello-Microverse
+
+This project is mainly about setting up diffrent development tools and working on gitHub workflows.
+
+## Built With
+
+- Major languages: HTML5,CSS
+- Technologies used: linters, Lighthouse, Webhint, Stylelint
+
+## Getting Started
```

Comment:
- [ ] You did a great job working on your project documentation 🚀 However, you're using an outdated version of the README template, please consider updating to [the new one](https://github.com/microverseinc/readme-template)! 🤓 🛠️